Assets/SteamVR_Unity_Toolkit/Scripts/VRTK_RoomExtender.cs

```diff
@@ -0,0 +1,46 @@
+using UnityEngine;
+using System.Collections;
+
+public class VRTK_RoomExtender : MonoBehaviour
+{
+	public float additionalMovementMultiplier = 1.0f;
+	public float headZoneRadius = 0.25f;
+	public Transform debugTransform;
+	public Transform movementTransform;
```

Comment:
Is this worth having as a parameter? You're always going to find it from the DeviceFinder anyway right?